apache/superset
# fix(home): Show home page tabs as pills instead of links

### SUMMARY
[home] pills show as links

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
![image](https://user-images.githubusercontent.com/47900232/171860391-e87c29fc-5d64-48e7-8db9-8a9ca5755818.png)

AFTER:
![Screen Shot 2022-06-03 at 9 11 49 AM](https://user-images.githubusercontent.com/47900232/171860813-12e1d570-cd98-4643-9029-6707024eb2df.png)

### TESTING INSTRUCTIONS
**How to reproduce this issue**
1. Go to Welcome page
2. Click or Hover the each tab
3. Should not display as link style when tab is selected

### ADDITIONAL INFORMATION
